staging/src/k8s.io/api/autoscaling/v2beta2/types.go

```diff
@@ -117,6 +120,26 @@ type MetricSpec struct {
 // MetricSourceType indicates the type of metric.
 type MetricSourceType string
 
+// HorizontalPodAutoscalerScaleConstraints configures the scaling velocity
+// by specifying the "absolute" value (in number of pods) and "relative" values (in percents)
+// All the parameters of the struct are "per minute".
+// For each scale direction (Up or Down) if both parameters are specified
+// the largest constraint is used.
+type HorizontalPodAutoscalerScaleConstraints struct {
+	// scaleUpPercent specifies the scale up relative speed, in percentages
+	// i.e. if scaleUpPercent = 150 , then we can add 150% more pods (10 -> 25 pods)
+	ScaleUpPercent *resource.Quantity `json:"scaleUpPercent,omitempty" protobuf:"bytes,1,opt,name=scaleUpPercent"`
```

Comment:
why Quantity?  That doesn't seem right.  Do you really want to be able to specify "0.1" or "3Mi" ?  Why is "int32" not sufficient?

Reply:
The idea behind quantity is that all values are per minute. So if you want to add/remove at most 1 pod every 2 minutes you need to put 0.5 here. 

Quantity is used for consistency with pods. We can however use ints in percent-like values if you prefer.